tensorflow/python/feature_column/feature_column_v2.py

```diff
@@ -2702,7 +2702,8 @@ def _transform_input_tensor(self, input_tensor):
           'SparseTensor is not supported. key: {}'.format(self.key))
     if self.normalizer_fn is not None:
       input_tensor = self.normalizer_fn(input_tensor)
-    return math_ops.to_float(input_tensor)
+    #return math_ops.to_float(input_tensor)
```

Comment:
Can you remove these comment lines?

Reply:
Thanks @rohan100jain , i have updated the code as per comments, kindly check.